clients/admin-ui/src/features/taxonomy/TaxonomyEntityTag.tsx

```diff
@@ -0,0 +1,28 @@
+import { Tag, TagCloseButton, TagLabel, TagProps } from "@fidesui/react";
+
+interface Props {
+  name: string;
+  onClose?: () => void;
+}
+const TaxonomyEntityTag = ({ name, onClose, ...other }: Props & TagProps) => {
+  const tagProps = {
+    backgroundColor: "primary.400",
+    color: "white",
+    "data-testid": `taxonomy-entity-${name}`,
+    width: "fit-content",
+    size: "sm",
+    ...other,
+  };
```

Comment:
This file is just renamed from `DataCategoryTag` since it can actually be used for more than just data categories. The bigger change is I pulled the props out into their own const so that they could be reused between the version of the tag that is just a tag and the one that has a close button